src/pages/journal/__tests__/journal.selector.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('JournalSelector', () => {
 
   describe('getSlots', () => {
     it('should select the test slots from the state', () => {
-      const selectedSlots = getSlotsOnSelectedDate(state);
+      const selectedSlots = getSlotsOnselectedDate(state);
```

Comment:
I preferred the previous casing.  Think it should still be getSlotsOnSelectedDate.

Reply:
Good spot. (find and replace - The Good Parts)